ci/scripts/move_to_production.py

```
@@ -0,0 +1,224 @@
+    data = {"query": query}
+    if variables:
+        data["variables"] = variables
+
+    response = requests.post("https://api.github.com/graphql",
+                           headers=headers,
+                           json=data)
+
+    if response.status_code != 200:
+        print(f"GraphQL request failed: {response.status_code}")
+        print(response.text)
+        exit(1)
```

Pylint (3.3.7)

[refactor] Consider using 'sys.exit' instead

(R1722)